src/Ledger/Conway/Conformance/Certs.agda

```diff
@@ -57,33 +57,36 @@ updateCertDeposit pp cert deposits
   = (deposits ∪⁺ certDeposit cert pp) ∣ certRefund cert ᶜ
 
 private variable
-  an                  : Anchor
-  dReps               : Credential ⇀ Epoch
-  pools               : KeyHash ⇀ PoolParams
-  retiring            : KeyHash ⇀ Epoch
-  vDelegs             : Credential ⇀ VDeleg
-  sDelegs stakeDelegs : Credential ⇀ KeyHash
-  rewards rwds        : Credential ⇀ Coin
-  dreps               : Credential ⇀ Epoch
-  voteDelegs          : Credential ⇀ VDeleg
-  ccKeys ccHotKeys    : Credential ⇀ Maybe Credential
-  dCert               : DCert
-  c                   : Credential
-  mc                  : Maybe Credential
-  d                   : Coin
-  e                   : Epoch
-  kh kh'              : KeyHash
-  mv                  : Maybe VDeleg
-  mkh                 : Maybe KeyHash
-  stᵍ stᵍ'            : GState
-  stᵈ stᵈ'            : DState
-  stᵖ stᵖ'            : PState
-  Γ                   : CertEnv
-  pp                  : PParams
-  dep gdep ddep edeps : Deposits
-  vs                  : List GovVote
-  poolParams          : PoolParams
-  wdrls               : RwdAddr ⇀ Coin
+  an : Anchor
+  dReps : Credential ⇀ Epoch
+  pools : KeyHash ⇀ PoolParams
+  vDelegs : Credential ⇀ VDeleg
+  sDelegs : Credential ⇀ KeyHash
+  retiring : KeyHash ⇀ Epoch
+  ccKeys : Credential ⇀ Maybe Credential
+  rwds : Credential ⇀ Coin
+  delegatees : ℙ Credential
+  dCert : DCert
+  c : Credential
+  mc : Maybe Credential
+  mv : Maybe VDeleg
+  d : Coin
+  e : Epoch
+  kh : KeyHash
+  mkh : Maybe KeyHash
+  stᵍ stᵍ' : GState
+  stᵈ stᵈ' : DState
+  stᵖ stᵖ' : PState
+  Γ : CertEnv
+  pp : PParams
+  dep ddep gdep : Deposits
+  vs : List GovVote
+  poolParams : PoolParams
+  wdrls  : RwdAddr ⇀ Coin
+  ccHotKeys : Credential ⇀ Maybe Credential
+  voteDelegs : Credential ⇀ VDeleg
+  stakeDelegs : Credential ⇀ KeyHash
+  rewards : Credential ⇀ Coin
```

Comment:
What happened to the nice alignment?

Reply:
Was there nice alignment here before?  I didn't remove any in this PR.  Anyway, I'll align this section nicely in the next, hopefully final, commit.